js/views/playingScreen.js

```diff
@@ -12,16 +12,22 @@ var React = require('react'),
     Watermark = require('../components/watermark'),
     ResizeMixin = require('../mixins/resizeMixin'),
     CONSTANTS = require('../constants/constants');
+    ViewControls = require('../components/viewControls');
 
 var PlayingScreen = React.createClass({
   mixins: [ResizeMixin],
 
   getInitialState: function() {
     this.isMobile = this.props.controller.state.isMobile;
     this.browserSupportsTouch = this.props.controller.state.browserSupportsTouch;
+
     return {
       controlBarVisible: true,
-      timer: null
+      timer: null,
+      isMouseDown: false,
+      isMouseMove: false,
+      XMouseStart: 0,
+      YMouseStart: 0,
```

Comment:
These should be `xMouseStart` and `yMouseStart` for consistent styling.

Reply:
\+